ServiceSamples/TLSConsoleApplication/Program.cs

```diff
@@ -0,0 +1,80 @@
+﻿using AuthenticationUtility;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Net.Security;
+using System.Web;
+using System.Security.Authentication;
+using System.Text;
+using System.Security.Cryptography.X509Certificates;
+
+namespace OAuthXppConsoleApplication
+{
+    class Program
+    {
+        public static string sessionUrl = "/api/services/UserSessionService/AifUserSessionService/GetUserSessionInfo";
+        
+        static void Main(string[] args)
+        {
+            string GetUserSessionOperationPath = string.Format("{0}{1}", ClientConfiguration.Default.UriString.TrimEnd('/'), sessionUrl);
+            
+            try
+            {
+                // Creates an HttpWebRequest for user session URL
+                HttpWebRequest aadRequest = (HttpWebRequest) WebRequest.Create(GetUserSessionOperationPath);
+
+                // Change TLS version of HTTP request if the TLS version value is defined in ClientConfiguration
+                if (!string.IsNullOrWhiteSpace(ClientConfiguration.OneBox.TLSVersion))
+                {
+                    aadRequest.ProtocolVersion = Version.Parse(ClientConfiguration.OneBox.TLSVersion);
+                }
+
+                string tlsRequestVersion = aadRequest.ProtocolVersion.ToString();                
+                Console.WriteLine("The TLS protocol version for the HTTP request is {0}.", tlsRequestVersion);
+                
+                aadRequest.Headers[OAuthHelper.OAuthHeader] = OAuthHelper.GetAuthenticationHeader();
+                aadRequest.Method = "POST";
+                aadRequest.ContentLength = 0;
+
+                // Get HttpWebResponse for the response
+                var aadResponse = (HttpWebResponse) aadRequest.GetResponse();
+
+                string tlsResponseVersion = aadResponse.ProtocolVersion.ToString();
+                Console.WriteLine("The TLS protocol version of the server response is {0}.", tlsResponseVersion);
+
+                if (aadResponse.StatusCode != HttpStatusCode.OK)
+                {
+                    Console.WriteLine("Could not get response from the server.");
+                    Console.ReadLine();
+                    return;
+                }
+
+                // Get response string
+                using (Stream responseStream = aadResponse.GetResponseStream())
+                {
+                    using (StreamReader streamReader = new StreamReader(responseStream))
+                    {
+                        string responseString = streamReader.ReadToEnd();
+
+                        Console.WriteLine(string.Format("Request sent using version {0}. Successfully received response with version {1}.", tlsRequestVersion, tlsResponseVersion));
```

Comment:
Do you also want to say which version we are talking about?

Reply:
If the authentication success then we know that the version that we defined in the request is compatible with the version in server.

Reply:
I was saying would it be better to say "Request sent using TLS version {0}" to be more specific.

Reply:
Updated.